code/modules/surgery/surgery_helpers.dm

```diff
@@ -22,6 +22,8 @@
 		return 0.85
 	if(drunk >= 15)//a little drunk
 		return 0.85
+	if(target.reagents.has_reagent("happiness")) // fuck yeah
+		return 0.81
```

Comment:
I bet someone will use this on their mentor application as an example of niche knowledge in like, 2 years if this PR gets merged